src/acl/external/AD_group/ext_ad_group_acl.cc

```diff
@@ -389,17 +390,18 @@ add_User_Group(wchar_t * Group)
     return 1;
 }
 
-/* returns 0 on match, -1 if no match */
-static int
-wccmparray(const wchar_t * str, const wchar_t ** array)
+/* returns true on match, false if no match */
+/* TODO: convert to std::containers */
+static bool
+wccmparray(const wchar_t * str, wchar_t ** array)
```

Comment:
I recommend renaming this function, so that the name matches its new API (and does not mislead callers). I suggest InArray(), but I do not insist on any specific name.

Reply:
> I can offer wStrIsInArray - the 'w' prefix is a common idiom in Windows for wide character functions

Yes, It was common to add such prefixes and suffixes for old type-unsafe languages and compilers. In a modern C++ context, these additions are usually just noise because the compiler does not let us supply the wrong type.